stable/aws-for-fluent-bit/templates/servicemonitor.yaml

```diff
@@ -29,6 +29,9 @@ spec:
 {{- if .Values.serviceMonitor.relabelings }}
     relabelings:
 {{ toYaml .Values.serviceMonitor.relabelings | indent 4 }}
+{{- end }}
+{{- with .Values.serviceMonitor.extraEndpoints }}
+    {{- toYaml . | nindent 2 }}
```

Comment:
Is this template correct?

I tried generating it, and it didn't work:

This was in values.yaml
```
  metricRelabelings: []
  extraEndpoints: []
      - port: metrics
        path: /metrics
        interval: 30s
        scrapeTimeout: 10s
        scheme: http
```

And this was the error output from template dry run:

```
Error: failed to parse stable/aws-for-fluent-bit/values.yaml: error converting YAML to JSON: yaml: line 387: did not find expected key
helm.go:84: [debug] error converting YAML to JSON: yaml: line 387: did not find expected key
failed to parse stable/aws-for-fluent-bit/values.yaml
helm.sh/helm/v3/pkg/cli/values.(*Options).MergeValues
	helm.sh/helm/v3/pkg/cli/values/options.go:55
main.runInstall
	helm.sh/helm/v3/cmd/helm/install.go:212
main.newTemplateCmd.func2
	helm.sh/helm/v3/cmd/helm/template.go:82
github.com/spf13/cobra.(*Command).execute
	github.com/spf13/cobra@v1.5.0/command.go:872
github.com/spf13/cobra.(*Command).ExecuteC
	github.com/spf13/cobra@v1.5.0/command.go:990
github.com/spf13/cobra.(*Command).Execute
	github.com/spf13/cobra@v1.5.0/command.go:918
main.main
	helm.sh/helm/v3/cmd/helm/helm.go:83
runtime.main
	runtime/proc.go:250
runtime.goexit
	runtime/asm_amd64.s:1594
```

Reply:
@PettitWesley  I have ammended the test intsrctions see below: the template is correct

Reply:
@PettitWesley I have also updated the chart version